### PR TITLE
Update to support parse server 3.0 with native JS promises

### PR DESCRIPTION
**Pull request by funkenstrahlen**
@flovilmart I tried to update the code to work with native JS promises. Can you please take a look at it? I am not very skilled with JS and therefore I'd feel better if you can check what I did.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "version": "0.1.1",
   "dependencies": {
     "gm": "~1.21.1",
-    "parse": "^1.3.5"
+    "parse": "^3.0"
   },
```
Comment (flovilmart) [inline package.json:9]: This should be 2.0 as 3.0 is not released in the JS SDK yet

**Review by flovilmart — CHANGES_REQUESTED** [review]
One small change :)

File: package.json (hunk 1 above)
Comment (funkenstrahlen) [reply package.json:9]: of course. done.

**Review by flovilmart — APPROVED** [review]
Thanks!

**Pull request merged**